pkg/rest/config_upload.go

```diff
@@ -194,20 +194,20 @@ func isApiDashboard(api api.Api) bool {
 
 func getExistingValuesFromEndpoint(client *http.Client, theApi api.Api, url string, apiToken string) (values []api.Value, err error) {
 
-	values = make([]api.Value, 0)
+	existingValues := make([]api.Value, 0)
```

Comment:
Nitpick: Actually there is no need to initialize `existingValues` here, since we only use `append` 
on the slice anyway. 

We can simply use:
```go
var existingValues []api.Value
```

(feel free to ignore this :laughing:)

Reply:
Hmm, I did now know that append works correctly on an uninitialized slice. I'll try it.